src/Employer/Employer.Web/Views/Dashboard/_NoVacanciesContent.cshtml

```diff
@@ -1,5 +1,8 @@
 ﻿@using Microsoft.Extensions.Options
 @inject IOptions<ExternalLinksConfiguration> _externalLinks
+@{ 
+    ViewBag.Vpv = "/page-no-vacancies";
```

Comment:
the naming of this file suggests that it is a partial. In which case should it have a vpv?

Reply:
i will remove this bit of code.